lib/lita/handlers/github_pinger.rb

```diff
@@ -116,6 +120,42 @@ def act_on_assign(body, response)
         response
       end
 
+      def act_on_pr_opened(body, response)
+        puts "Detected that someone opened a #{type.tr('_', ' ')}."
+        if type.nil?
+          puts 'Neither pull request or issue detected, exiting...'
+          return
+        end
+
+        if config.enable_round_robin
+          puts "round robin is enabled, selecting the next engineer.."
+          next_reviewer_redis_key = 'lita-github-pinger:nexteng'
```

Comment:
what defines `nexteng`?

Reply:
@alekhinen so this is just a unique key for redis so that we don't overwrite any other data. This string could be anything that wouldn't conflict with anything in the DB

Reply:
ah yep this makes sense now.